### PR TITLE
[HEALTHVIC-1187] - added table of content to sitemap

### PR DESCRIPTION
**Pull request by awset**
## Motivation and Context

**JIRA issue:** https://salsadigital.atlassian.net/browse/HEALTHVIC-1187

## Changed

1. added table of content to ripple sitemap
2. can be toggle on and off based on the value from site term in Tide.

### Screenshots
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Improvement/refactoring (non-breaking change that doesn't add any features but makes things better)

## Checklist

- [x] I've added relevant changes to the documentation.
- [ ] I have added tests to cover my changes (if not applicable, please state why)
- [ ] My change requires a template update for create-ripple-app.
- [ ] I have added template update script for next release.

### CODE REVIEW

**Review by gargsuchi — DISMISSED** [review]

**Comment by MdNadimHossain** [comment]
Testing passed and it is in the 1.41.0 release list, so needs the review for this please

File: examples/reference/nuxt.config.js [hunk 1]
```diff
@@ -144,7 +144,7 @@ export default {
       username: process.env.CONTENT_API_AUTH_USER,
       password: process.env.CONTENT_API_AUTH_PASS
     },
-    site: 8888,
+    site: 4,
```
Comment (MdNadimHossain) [inline examples/reference/nuxt.config.js:147]: the test branch was old and it was in content-vic, that is why it needed to point to that. 
Revert back before merging

Comment (dylankelly) [reply examples/reference/nuxt.config.js:147]: @MdNadimHossain prefer that this is done in Lagoon CLI these days FYI

**Review by dylankelly — DISMISSED** [review]

File: examples/reference/nuxt.config.js (hunk 1 above)
Comment (MdNadimHossain) [reply examples/reference/nuxt.config.js:147]: hey @dylankelly 
I don't see any vars in the CMDB though - https://github.com/dpc-sdp/sdp-cmdb/blob/master/config/ripple/config.yml
Will raise it with devops if we can manage it from there may be :/

**Review by MdNadimHossain — APPROVED** [review]

**Pull request merged**